presentations/scopes/index.html

```diff
@@ -101,8 +101,7 @@
   <p>
   $$
     \begin{align}
-    X &\sqsubseteq Y \quad \text{// Read as } Y \text{ satisfies } X \\
-    &\phantom{\sqsubseteq \{} \text{// Derive tighter scope-sets} \\
```

Comment:
Also, to the extent the title makes sense, I think it's summarized by the line I added at the end.